bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/ResolutionMode.java

```diff
@@ -3,38 +3,25 @@
 import edu.gemini.spModel.type.DisplayableSpType;
 
 import java.util.*;
+import java.util.stream.Collectors;
 
 public enum ResolutionMode implements DisplayableSpType {
     // This will be used by every instrument other than GHOST and thus can be considered disjoint;
     // the overlapping name should not make a difference.
-    Standard("standard", "Standard Resolution", Collections.emptyList()),
-
-    GhostStandard("ghostStandard", "Standard Resolution",
-            Arrays.asList(
-                    AsterismType.GhostSingleTarget,
-                    AsterismType.GhostDualTarget,
-                    AsterismType.GhostTargetPlusSky,
-                    AsterismType.GhostSkyPlusTarget)
-    ),
-    GhostHigh("ghostHigh", "High Resolution",
-            Collections.singletonList(
-                    AsterismType.GhostHighResolutionTargetPlusSky
-            )),
-    GhostPRV("ghostPRV", "Precision Radial Velocity",
-            Collections.singletonList(
-                    AsterismType.GhostHighResolutionTargetPlusSky
-            ));
+    Standard("standard", "Standard Resolution"),
```

Comment:
What's the difference between `Standard` and `GhostStandard`? Or is `Standard` legacy now?

Reply:
I don't know, that's a good question.  I assume it just distinguishes GHOST from the others.